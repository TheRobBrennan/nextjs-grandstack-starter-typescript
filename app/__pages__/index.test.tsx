import React from "react"
import TestRenderer from "react-test-renderer"

import DefaultPage from "../pages/index"
import { MockedProvider } from "@apollo/client/testing"

describe("The default page", () => {
  it(`should render`, () => {
    // Verify success state
    const component = TestRenderer.create(
      <MockedProvider mocks={[]} addTypename={false}>
        <DefaultPage />
      </MockedProvider>
    )
    expect(component.toJSON()).toMatchSnapshot()
  })
})
