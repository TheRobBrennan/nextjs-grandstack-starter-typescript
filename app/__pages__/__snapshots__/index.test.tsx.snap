// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The default page should render 1`] = `
<div
  className="makeStyles-root-1"
>
  <header
    className="MuiPaper-root MuiAppBar-root MuiAppBar-positionAbsolute MuiAppBar-colorPrimary makeStyles-appBar-6 MuiPaper-elevation4"
  >
    <div
      className="MuiToolbar-root MuiToolbar-regular makeStyles-toolbar-5 MuiToolbar-gutters"
    >
      <img
        alt="GRANDstack logo"
        className="makeStyles-appBarImage-8"
        href="/"
        onClick={[Function]}
        onMouseEnter={[Function]}
        src="img/grandstack.png"
      />
      <h1
        className="MuiTypography-root makeStyles-title-7 MuiTypography-h6 MuiTypography-colorInherit MuiTypography-noWrap"
      >
        Next.js GRANDstack Starter with @neo4j/graphql ^1.0.0-alpha.2 and TypeScript
      </h1>
    </div>
  </header>
  <main
    className="makeStyles-content-3"
  >
    <div
      className="makeStyles-appBarSpacer-2"
    />
    <div
      className="MuiContainer-root makeStyles-container-4 MuiContainer-maxWidthLg"
    >
      <div
        className="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-4"
      >
        <div
          className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-md-8 MuiGrid-grid-lg-7"
        >
          <div
            className="MuiPaper-root makeStyles-paper-10 makeStyles-fixedHeight-11 MuiPaper-elevation1 MuiPaper-rounded"
          >
            <p>
              Loading
            </p>
          </div>
        </div>
        <div
          className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-md-4 MuiGrid-grid-lg-5"
        >
          <div
            className="MuiPaper-root makeStyles-paper-10 makeStyles-fixedHeight-11 MuiPaper-elevation1 MuiPaper-rounded"
          >
            <h2
              className="MuiTypography-root MuiTypography-h6 MuiTypography-colorPrimary MuiTypography-gutterBottom"
            >
              Total Users
            </h2>
            <p
              className="MuiTypography-root MuiTypography-h4"
            >
              Loading...
            </p>
            <p
              className="MuiTypography-root makeStyles-depositContext-12 MuiTypography-body1 MuiTypography-colorTextSecondary"
            >
              users found
            </p>
          </div>
        </div>
        <div
          className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12"
        >
          <div
            className="MuiPaper-root makeStyles-paper-10 MuiPaper-elevation1 MuiPaper-rounded"
          >
            <p>
              Loading
            </p>
          </div>
        </div>
      </div>
      <div
        className="MuiBox-root MuiBox-root-14"
      >
        <p
          className="MuiTypography-root MuiTypography-body2 MuiTypography-colorTextSecondary MuiTypography-alignCenter"
        >
          Copyright Â© 
          2021
           
          <a
            className="MuiTypography-root MuiLink-root MuiLink-underlineHover MuiTypography-colorInherit"
            href="/"
            onBlur={[Function]}
            onFocus={[Function]}
          >
            Robert J Brennan
          </a>
          . All rights reserved.
        </p>
      </div>
    </div>
  </main>
</div>
`;
